usagov/vote-gov-drupal
# VOTE-2017 remove hmn configuration

## Jira ticket

https://cm-jira.usa.gov/browse/VOTE-2017

## Description

Remove Hmong translation from configuration and string translations

## Deployment and testing

### Post-deploy steps
**If you have trouble running this locally it's because there is hmong content in the system. You may have to delete the hmong content first or delete the hmong translation in the admin interface** 

1. run `lando retune`

### QA/Testing instructions

1. log in and visit http://vote-gov.lndo.site/admin/config/regional/language
2. Confirm Hmong is not listed anymore in the languages list
3. visit http://vote-gov.lndo.site/about-us
4. Confirm Hmong is not listed in the language selector

## Checklist for the Developer




## Checklist for the Peer Reviewers
